db/seeds.rb

```diff
@@ -5,3 +5,20 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+10.times do
+  Term.create(name: Faker::Hipster.word)
```

Comment:
it's less funny and more it was an easy one for a word. THERE ARE TOO MANY FAKERS NOW UGHHHHHH